### PR TITLE
Govbar logo added to header

### PR DESCRIPTION
**Pull request by petzlux**

### CODE REVIEW

**Comment by petzlux** [comment]
@pgiraud care to review ? or too trivial ?

**Comment by elemoine** [comment]
Out of curiosity, why did you change the indentation? We use 2 spaces everywhere else, so why doing differently for index.html?

**Comment by petzlux** [comment]
okay my mistake , bad vim config, will revert

File: geoportailv3/templates/index.html [hunk 1]
```diff
@@ -22,8 +22,10 @@
     <!-- Begin fixed top bar -->
     <header class="navbar navbar-default navbar-fixed-top" role="navigation">
       <div class="container-fluid">
-        <div class="">
-          <a href class="navbar-brand" translate>logo</a>
+        <div class="navbar-inner">
+          <a href class="pull-left brand" translate>
```
Comment (elemoine) [inline geoportailv3/templates/index.html]: What does the `translate` directive translate here?

**Comment by elemoine** [comment]
This looks good to me. I've just wondered about the `translate` directive.

**Comment by petzlux** [comment]
will change that, oversight from me

**Comment by petzlux** [comment]
@elemoine why did one of two travis builds fail ? complains about problems in ngeo module ?

**Comment by elemoine** [comment]
I think it's related to https://github.com/Geoportail-Luxembourg/geoportailv3/pull/442. Could you please try to rebase your branch?

**Pull request merged**